back-end/www/models/model_operations/user_operations.py

```diff
@@ -129,3 +129,44 @@ def remove_user(user_id):
 
     db.session.delete(user)
     db.session.commit()
+
+
+def get_user_count():
+    """
+    Get total user count.
+
+    Returns
+    -------
+    count : int
+        Number of total users.
+    """
+    count = User.query.count()
+
+    return count
+
+
+def get_user_done_location_count(user_id):
+    """
+    Get the distinct location count that the user has ever identified successfully.
+
+    Parameters
+    ----------
+    user_id : int
+        ID of the user.
+
+    Returns
+    -------
+    count : int
+        Number of locations that the user identified.
+    """
+    user = get_user_by_id(user_id)
+
+    if user is None:
+        raise Exception("Cannot find the user.")
+
+    if user.answers is None:
+        return 0
+
+    loc_count = len(set([answer.location_id for answer in user.answers]))
```

Comment:
Should there be a check to see if the done_at column in the location table is not None? So that we know the location has been identified? It is possible that a user provides an answer, but the answer has not been verified by other users, right?

Reply:
This has been discussed last week. They said they don't want "precise statistics", but "beautiful statistics" instead to encourage the users. Therefore, all the answers provided by the user should take effect when calculating this number. 
@yalgorithm777

Reply:
I am sorry that I didn't write a clear spec.

`loc_count` needs to check if the done_at column is not None. We need this precise statistics for internal calculation.

As for the "beautiful statistics", I referred to the count of `user.answers` that we want to show how many answers we have collected to encourage users, not excluding answers that didn't pass the gold standard.

Reply:
Because I wrote this function to implement the API status/individual_answer_count, so....I think I need to change the name not to confuse anybody. Maybe we need to define more statistics API to get precise internal calculation?